wire-schema/src/jvmTest/kotlin/com/squareup/wire/schema/PrunerTest.kt

```diff
@@ -158,13 +158,349 @@ class PrunerTest {
           |message TestMessage {
           |  oneof element {
           |    option (my_custom_oneOf_option) = true;
-          |  
+          | $space
```

Comment:
nit: is it a bug to have a space here? Seems like we're just printing indent before an empty line?

Reply:
That's because we're in the context of the `oneof`. Cannot really know what's before us.